src/config/__tests__/config.test.ts

```diff
@@ -0,0 +1,133 @@
+import { ETHEREUM_NETWORK } from 'src/config/networks/network.d'
+
+describe('Config Services', () => {
+  beforeEach(() => {
+    jest.resetModules()
+  })
+
+  it(`should load 'test' network config`, () => {
+    // Given
+    jest.mock('src/utils/constants', () => ({
+      NODE_ENV: 'test',
+    }))
+    const { getNetworkInfo } = require('src/config')
+
+    // When
+    const networkInfo = getNetworkInfo()
+
+    // Then
+    expect(networkInfo.id).toBe(ETHEREUM_NETWORK.LOCAL)
+  })
+
+  it(`should load 'mainnet' network config`, () => {
+    // Given
+    jest.mock('src/utils/constants', () => ({
+      NODE_ENV: '',
+      NETWORK: 'MAINNET',
+    }))
+    const { getNetworkInfo } = require('src/config')
+
+    // When
+    const networkInfo = getNetworkInfo()
+
+    // Then
+    expect(networkInfo.id).toBe(ETHEREUM_NETWORK.MAINNET)
+  })
+
+  it(`should load 'mainnet.dev' network config`, () => {
+    // Given
+    jest.mock('src/utils/constants', () => ({
+      NODE_ENV: '',
+      NETWORK: 'MAINNET',
+    }))
+    const { getTxServiceUrl, getGnosisSafeAppsUrl } = require('src/config')
+    const TX_SERVICE_URL = 'https://safe-transaction.mainnet.staging.gnosisdev.com/api/v1/'
```

Comment:
shouldn't it be taken from `src/config/networks/mainnet.ts`?

Reply:
:thinking: yes, you're right, as the goal of the test is to be sure that `NODE_ENV` and `NETWORK` are valid.